releases/release-1.31/release-notes/maps/pr-119589-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 119589
+releasenote:
+  text: |-
+    kubectl support both:
```

```suggestion
    Added kubectl support for:
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653150748